weather_dl_v2/cli/README.md

```diff
@@ -46,10 +46,14 @@ gcloud compute instances create-with-container weather-dl-v2-cli \
     --boot-disk-device-name=weather-dl-v2-cli \
     --container-image=$IMAGE_PATH \
     --container-restart-policy=on-failure \
+    --container-tty \
     --no-shielded-secure-boot \
     --shielded-vtpm \
-    --shielded-integrity-monitoring \
-    --labels=goog-ec-src=vm_add-gcloud,container-vm=cos-stable-105-17412-101-24
+    --labels=goog-ec-src=vm_add-gcloud,container-vm=cos-stable-105-17412-101-24 \
+    --metadata=startup-script='#! /bin/bash
+    command="docker exec -it \\\$(docker ps -qf name=weather-dl-v2-cli) /bin/bash"
+    sudo sh -c "echo \"$command\" >> /etc/profile"
+    '
 ```
 
 ## To get the weather-dl-v2 cli docker-image id in VM
```

Comment:
Please remove this as well.

Reply:
updated in [03005d2](https://github.com/google/weather-tools/pull/364/commits/03005d290316f1adf10995ec605cf34d31ccff18)